src/card.tsx

```diff
@@ -393,3 +406,252 @@ export const SnapCard = React.forwardRef<HTMLDivElement, SnapCardProps>(
         );
     }
 );
+
+type CardAction = {
+    label: string;
+    onPress: () => void;
+    Icon: React.FC<IconProps>;
+};
+
+type CardActionsGroupProps = {
+    actions: Array<CardAction>;
+    isInverse?: boolean;
+};
+
+const CardActionsGroup = ({actions, isInverse}: CardActionsGroupProps): JSX.Element => {
+    return (
+        <Inline space={0}>
+            {actions.map(({onPress, label, Icon}, index) => (
+                <IconButton
+                    size={48}
+                    key={index}
+                    onPress={onPress}
+                    aria-label={label}
+                    className={styles.cardActionIconButton}
+                    style={{display: 'flex'}}
+                >
+                    <div className={isInverse ? styles.cardActionInverse : styles.cardAction}>
+                        <Icon color={vars.colors.neutralHigh} />
+                    </div>
+                </IconButton>
+            ))}
+        </Inline>
+    );
+};
+
+type MaybeWithActionsProps = {
+    children: React.ReactNode;
+    width?: string;
+    actions?: Array<CardAction>;
+    onClose?: () => void;
+    isInverse?: boolean;
+    'aria-label'?: string;
+};
+
+const MaybeWithActions = ({
+    children,
+    width,
+    actions,
+    onClose,
+    isInverse,
+    'aria-label': ariaLabel,
+}: MaybeWithActionsProps): JSX.Element => {
+    const {texts} = useTheme();
+    const finalActions = actions ? [...actions] : [];
+    if (onClose) {
+        finalActions.push({
+            label: texts.closeButtonLabel,
+            onPress: onClose,
+            Icon: IconCloseRegular,
+        });
+    }
+
+    if (!finalActions.length) {
+        return <>{children}</>;
+    }
+
+    return (
+        <section aria-label={ariaLabel} style={{width: width || '100%', position: 'relative'}}>
+            <HasActionsContext.Provider value>{children}</HasActionsContext.Provider>
+            <div style={{position: 'absolute', right: 8, top: 8}}>
+                <CardActionsGroup actions={finalActions} isInverse={isInverse} />
+            </div>
+        </section>
+    );
+};
+
+interface CommonDisplayCardProps {
+    /**
+     * Typically a mistica-icons component element
+     */
+    icon?: React.ReactElement;
+    actions?: Array<CardAction>;
+    onClose?: () => void;
+    dataAttributes?: DataAttributes;
+    headline?: React.ReactComponentElement<typeof Tag>;
+    pretitle?: string;
+    pretitleLinesMax?: number;
+    title: string;
+    titleLinesMax?: number;
```

Comment:
we don't allow this `xLinexMax`in Hero component becuase design told us this is an a11y issue...  @yceballost ?

Reply:
but we need this for some Cards configured by CMS

Reply:
CMS need this control . Ideally, this prop wouldn't have to exist, at least, in the title. In hero we didn't add it to avoid these AA problems, it is possible in the future we need to include this lines limitation in hero, but we will wait for this need.

Reply:
why "CMS need" this? if this is really an accessibility issue, this should be avoid and CMS operator be aware of this, dont know why this is needed because of the CMS